### PR TITLE
[VI] Add `Registration`

### PR DESCRIPTION
**Pull request by ltln**
There are some phrases that are not translated in-game so I keep it original.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Registration/vi.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Đăng ký
+
+*Lưu ý: Có hơn một tài khoản osu! là vi phạm [nội quy osu!](/wiki/Rules)!*
+
+1. Sau khi [cài đặt](/wiki/Client/Installation) và chạy osu!, osu! sẽ yêu cầu bạn đăng nhập hoặc đăng ký. Giả sử bạn đang ở đây vì cần một tài khoản, nhấp vào `Create an account`.
+
+   ![Màn hình đăng ký trong game](img/ingame-registration.jpg "Màn hình đăng ký trong game")
+
+2. Điền vào tất cả các ô trong biểu mẫu đăng ký.
+   - Về ô username, bạn có thể đặt username tối đa 15 ký tự; bao gồm các ký tự (`a-z`, `A-Z`, `0-9`), gạch dưới (`_`), ngoặc vuông, (`[` and `]`), gạch ngang (`-`), và dấu cách (` `).
```
Comment (TranT3011) [inline wiki/Registration/vi.md:10]:
```suggestion
   - Về ô username, bạn có thể đặt tối đa 15 ký tự; bao gồm các ký tự (`a-z`, `A-Z`, `0-9`), gạch dưới (`_`), ngoặc vuông, (`[` and `]`), gạch ngang (`-`), và dấu cách (` `)
```
repeating the word sounds weird to me

File: wiki/Registration/vi.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+   - Về ô email address, hãy dùng địa chỉ email của bạn. Email này sẽ dùng để đặt lại mật khẩu cho tài khoản khi bạn quên và gửi mã xác nhận cho bạn khi cần thiết, cho nên đừng dùng email rác.
+   - Về ô password, hãy dùng mật khẩu dài ít nhất 8 ký tự.
```
Comment (TranT3011) [inline wiki/Registration/vi.md:12]:
```suggestion
   - Về ô password, hãy đặt ít nhất 8 ký tự.
```
make it sound more natrual

File: wiki/Registration/vi.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+3. Sau khi xong, nhấp vào `1. Create my account!`. Bạn sẽ được tự động đăng nhập vào.
+
+## Làm gì tiếp theo?
+
+Đừng quên đọc thật thật là kỹ [nội quy](/wiki/Rules)!
```
Comment (TranT3011) [inline wiki/Registration/vi.md:18]:
```suggestion
Đừng quên đọc [nội quy](/wiki/Rules) thật là kỹ !
```
same as 12

**Review by TranT3011 — CHANGES_REQUESTED** [review]

**Review by TranT3011 — DISMISSED** [review]
im vnese (https://osu.ppy.sh/users/19233714)
the translation is good
dont know how to make this approval sound better lol

**Review by TranT3011 — APPROVED** [review]
a

**Review by TPGPL — APPROVED** [review]

**Pull request merged**